File: examples/v2/organization-members/go/main.go
package main

import (
	v2 "github.com/pulumi/pulumi-pulumiservice/sdk/go/pulumiservice/v2"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		cfg := config.New(ctx, "")
		organizationName := cfg.Get("organizationName")
		if organizationName == "" {
			organizationName = "service-provider-test-org"
		}
		memberLogin := cfg.Get("memberLogin")
		if memberLogin == "" {
			memberLogin = "pulumi-bot"
		}
		memberRole := cfg.Get("memberRole")
		if memberRole == "" {
			memberRole = "member"
		}

		member, err := v2.NewOrganizationMember(ctx, "member", &v2.OrganizationMemberArgs{
			OrgName:   pulumi.String(organizationName),
			UserLogin: pulumi.String(memberLogin),
			Role:      pulumi.String(memberRole),
		})
		if err != nil {
			return err
		}

		ctx.Export("memberId", member.ID())
		return nil
	})
}

File: examples/v2/organization-members/python/__main__.py
import pulumi
import pulumi_pulumiservice.v2 as ps_v2

config = pulumi.Config()
service_org = config.get("serviceOrg") or "service-provider-test-org"
member_login = config.get("memberLogin") or "pulumi-bot"
member_role = config.get("memberRole") or "member"

member = ps_v2.OrganizationMember(
    "member",
    org_name=service_org,
    user_login=member_login,
    role=member_role,
)

pulumi.export("memberId", member.id)
